# Add Message header
Header header

# Consider one object now, but how to deal with multiple object?
float64    object_distance
float64    object_direction
float64    object_speed 
float64    object_orientation 

#Back up file.
